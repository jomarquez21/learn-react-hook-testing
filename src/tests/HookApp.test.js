import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import {HookApp} from '../HookApp';

describe('test on `HookApp`', () => {
  test('snapshot from `HookApp`', () => {
    const component = shallow(<HookApp />);

    expect(toJson(component)).toMatchSnapshot();
  });
});