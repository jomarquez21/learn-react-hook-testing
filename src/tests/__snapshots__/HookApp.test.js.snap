// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on \`HookApp\` snapshot from \`HookApp\` 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
